otterai/indexer.py

```diff
@@ -30,8 +30,20 @@ def get_file_type(file_path: str) -> Optional[str]:
         return 'frontend'
     elif ext in ['.sql', '.graphql']:
         return 'data'
-    elif ext in ['.test.js', '.test.ts', '.spec.py', '_test.go']:
+    elif ext in ['.test.js', '.test.ts', '.spec.py', '_test.go', '.spec.js', '.spec.ts']:
         return 'test'
+    elif ext in ['.env', '.env.*']:
+        return 'environment'
+    elif ext in ['.gitignore', '.dockerignore']:
```

Comment:
🗑️ The check for '.gitignore' and '.dockerignore' is good, but ensure that the handling of these files does not expose sensitive data. 🔍